ws2812b/src/test/java/com/google/android/things/contrib/driver/ws2812b/ColorChannelSequenceTest.java

```diff
@@ -0,0 +1,100 @@
+package com.google.android.things.contrib.driver.ws2812b;
+
+
+import android.graphics.Color;
+
+import com.google.android.things.contrib.driver.ws2812b.util.ColorMock;
+import com.google.android.things.contrib.driver.ws2812b.util.ColorUtil;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(android.graphics.Color.class)
+public class ColorChannelSequenceTest {
```

Comment:
nice tests 👍

Reply:
Greater thanks should go to the author of the apa102 driver: https://github.com/androidthings/contrib-drivers/blob/master/apa102/src/test/java/com/google/android/things/contrib/driver/apa102/ColorMock.java ;)